src/main.ts

```diff
@@ -15,10 +15,13 @@ const Config = {
     | "production"
     | "development",
   vitePort: 5173,
+  viteServerSecure: false,
 };
 
 function getViteHost() {
-  return `http://localhost:${Config.vitePort}`;
+  return `http${Config.viteServerSecure ? "s" : ""}://localhost:${
```

Comment:
I would prefer to fully utilise the ternary
```suggestion
  return `${Config.viteServerSecure ? "https" : "http"}://localhost:${
```
Other that that looks great!